src/httprpc.cpp

```diff
@@ -226,6 +226,11 @@ static bool InitRPCAuthentication()
     return true;
 }
 
+void RegisterJSONEndpoint(const std::string& endpoint, bool exactMatch)
```

Comment:
a) I think we don't want an `#ifdef ENABLE_WALLE` in `httpserver.cpp`
b) Using `RegisterHTTPHandler(endpoint, exactMatch, HTTPReq_JSONRPC);` from the point where we can register based on RPC-tables endpoints would result in exposing `RegisterHTTPHandler` and `HTTPReq_JSONRPC` which seems unideal.

But the idea of register based on the tables endpoints makes sense, will implement but will also keep the `RegisterJSONEndpoint` function.